tests/DependencyInjection/SonataUserExtensionTest.php

```diff
@@ -226,6 +226,28 @@ public function testConfigureGoogleAuthenticatorEnabled(): void
         $this->assertContainerBuilderHasServiceDefinitionWithArgument('sonata.user.google.authenticator.provider', 0, 'bar');
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The 'Google\Authenticator' namespace is deprecated in sonata-project/GoogleAuthenticator since version 2.1 and will be removed in 3.0.
```

Comment:
It is weird that you need to do this :sweat_smile:

Reply:
Yes, it also seems weird to me, but this deprecation is risen everytime the `load` method is used :/